coder/main.go

```diff
@@ -201,6 +246,12 @@ func Run(s *discordgo.Session, m *discordgo.MessageCreate) error {
 	s.MessageReactionAdd(m.ChannelID, m.ID, "✅")
 
 	lines := strings.Join(code.Lines, "\n")
+	
+	if len(lines)> discordCharLimit {
+		//will only happen on file input
+		lines = "Input to long to parse :(\n"
+	}
```

Comment:
I think this should be thrown as an error instead of manipulating `lines`. Create a new error construct as on lines 21-22 and return that (such as "TooLongInputCode").

Reply:
By not throwing an error here it allows the user to post a longer input file, and as long as the output message is less then the discord chat limit the program would still run